napari/layers/points/points.py

```diff
@@ -1057,7 +1103,7 @@ def _get_state(self):
                 'properties': self.properties,
                 'property_choices': self.property_choices,
                 'text': self.text.dict(),
-                'n_dimensional': self.n_dimensional,
```

Comment:
I think we need to bring this back for the reasons in this comment: https://github.com/napari/napari/pull/4007#issuecomment-1023271956

Reply:
My guess it's unlikely that any plugin relies on `n_dimensional`, but probably better to be safe than sorry?